hieradata/org/ncsa.yaml

```diff
@@ -13,6 +13,13 @@ baseline_cfg::networkmanager::ensure: "running"
 
 chrony::leapsectz: "right/UTC"
 
+docker::overlay2_override_kernel_check: true
```

Comment:
Non-blocker: we're copy/pasting these values in a lot of place, I'd like to make them defaults. Thoughts?

Reply:
The module level hiera data for the profile would be a good place to make them defaults. I'm okay with that.
I'm happy to put those in if there is agreement.

Reply:
nevermind, it can't go in module level hiera data (as is). It would require changing the profile to take parameters and then set defaults for those parameters in module level hiera. I'll proceed with the merge as is and leave this topic for a future change.